lte/gateway/python/integ_tests/defs.mk

```diff
@@ -294,6 +293,7 @@ s1aptests/test_restore_config_after_non_sanity.py
 # s1aptests/test_ipv4v6_non_nat_ded_bearer_dl_tcp.py
 # s1aptests/test_ipv6_non_nat_ded_bearer_ul_tcp.py
 # s1aptests/test_ipv6_non_nat_ded_bearer_dl_tcp.py
+# s1aptests/test_ipv6_non_nat_dp_ul_tcp.py
```

Comment:
Nitpick: Might fit better with the set of tests above (line 285-87).

Reply:
:+1: moved